src/collection-view.js

```diff
@@ -99,7 +99,7 @@ const CollectionView = Backbone.View.extend({
   _initialEvents() {
     if (this.collection) {
       this.listenTo(this.collection, 'add', this._onCollectionAdd);
```

Comment:
thats for render though, not update add. I am still trying to figure out how to deal with that because `options.at` is screwing up how I think the views should update and render. I might do a broken PR push just to give you all a chance to see what I am talking about.